src/components/TodoList/TodoList.tsx

```diff
@@ -14,195 +33,53 @@ export const TodoList: React.FC = () => {
             <th>#</th>
```

Comment:
There is constant There are no todos matching current filter criteria, which should only show up when it's really like this